<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>空教室查询系统主界面</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../image/searchIcon.png</normaloff>../image/searchIcon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#frame_background{
border-image:url(:/image/background-3.png);
}

#frame_login{background-color:#ffffff}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="frame_background">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="Btn_return">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>45</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#Btn_return {
        background-color: rgb(255, 138, 140);
		border: 2px groove gray;
        border-radius: 10px;
 		padding: 2px 4px;
        border-style: outset;
    }

    QPushButton#Btn_return:hover {
        background-color: rgb(229, 241, 251);
        color: black;
    }

    QPushButton#Btn_return:pressed {
        background-color: rgb(204, 228, 247);
        border-style: inset;
    }

QPushButton#Btn_return {
        color: blue;
        font-weight: bold;
        text-stroke: 1px black;
    }
</string>
          </property>
          <property name="text">
           <string>返回</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>请选择你要反馈的教室进行修改</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="C1">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>45</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#C1 {
        background-color: rgb(255, 138, 140);
		border: 2px groove gray;
        border-radius: 10px;
 		padding: 2px 4px;
        border-style: outset;
    }

    QPushButton#C1:hover {
        background-color: rgb(229, 241, 251);
    }


QPushButton#C1 {
        color: blue;
        font-weight: bold;
        text-stroke: 1px black;
    }

QPushButton#C1:checked {
        background-color: rgb(170, 0, 0);
    }</string>
          </property>
          <property name="text">
           <string>教1</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_20">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="C2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>45</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#C2 {
        background-color: rgb(255, 138, 140);
		border: 2px groove gray;
        border-radius: 10px;
 		padding: 2px 4px;
        border-style: outset;
    }

    QPushButton#C2:hover {
        background-color: rgb(229, 241, 251);
    }


QPushButton#C2 {
        color: blue;
        font-weight: bold;
        text-stroke: 1px black;
    }

QPushButton#C2:checked {
        background-color: rgb(170, 0, 0);
    }</string>
          </property>
          <property name="text">
           <string>教2</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_29">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="C3">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>45</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#C3 {
        background-color: rgb(255, 138, 140);
		border: 2px groove gray;
        border-radius: 10px;
 		padding: 2px 4px;
        border-style: outset;
    }

    QPushButton#C3:hover {
        background-color: rgb(229, 241, 251);
    }


QPushButton#C3 {
        color: blue;
        font-weight: bold;
        text-stroke: 1px black;
    }

QPushButton#C3:checked {
        background-color: rgb(170, 0, 0);
    }</string>
          </property>
          <property name="text">
           <string>教3</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_30">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="C4">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>45</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#C4 {
        background-color: rgb(255, 138, 140);
		border: 2px groove gray;
        border-radius: 10px;
 		padding: 2px 4px;
        border-style: outset;
    }

    QPushButton#C4:hover {
        background-color: rgb(229, 241, 251);
    }


QPushButton#C4 {
        color: blue;
        font-weight: bold;
        text-stroke: 1px black;
    }

QPushButton#C4:checked {
        background-color: rgb(170, 0, 0);
    }</string>
          </property>
          <property name="text">
           <string>教4</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <spacer name="verticalSpacer_9">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <spacer name="horizontalSpacer_17">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QFrame" name="frame_pic">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>500</width>
                   <height>600</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <widget class="QTableWidget" name="tableWidget">
                  <property name="geometry">
                   <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>521</width>
                    <height>600</height>
                   </rect>
                  </property>
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>400</width>
                    <height>600</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>600</width>
                    <height>600</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">#tableWidget{ background-color: #aaaaff; }
/* 设置表格的基本样式 */  
QTableView {  
border-image:url(:/res/pic/background-1.png);
    background-color: #f9f9f9; /* 背景色 */  
    border: 2px solid #a1a1a1; /* 边框 */  
    gridline-color: #e0e0e0; /* 网格线颜色 */  
    selection-background-color: #88d8b0; /* 选中项背景色 */  
    selection-color: #ffffff; /* 选中项文字颜色 */  
}  
  
/* 设置表头样式 */  
QHeaderView::section {  
    background-color: #f0a3a3; /* 背景色 */  
    color: #ffffff; /* 文字颜色 */  
    border: 1px solid #a1a1a1; /* 边框 */  
    border-bottom: 2px solid #a1a1a1; /* 底边框加粗 */  
    padding: 4px; /* 内边距 */  
    font-weight: bold; /* 字体加粗 */  
}  
  
/* 设置表格项的样式 */  
QTableView::item {  
    border: none; /* 无边框 */  
    padding: 4px; /* 内边距 */  
	background-color: transparent; 
}  
  
/* 奇数行样式 */  
QTableView::item:nth-child(odd) {  
    background-color: #e9e9e9; /* 奇数行背景色 */  
}  
  
/* 偶数行样式 */  
QTableView::item:nth-child(even) {  
    background-color: #f5f5f5; /* 偶数行背景色 */  
}  
  
/* 鼠标悬停时的样式 */  
QTableView::item:hover {  
    background-color: #c8e6c9; /* 鼠标悬停时的背景色 */  
}  
  
/* 选中项的样式（当与selection-background-color不同时使用）*/  
QTableView::item:selected {  
    background-color: #a5d6a7; /* 选中项背景色 */  
}  
  
/* 自定义滚动条样式 */  
QScrollBar:vertical {  
    border: none; /* 无边框 */  
    background: #f0f0f0; /* 背景色 */  
    width: 15px; /* 宽度 */  
    margin: 0px 0px 0px 0px; /* 边距 */  
}  
  
QScrollBar::handle:vertical {  
    background: #a8a8a8; /* 滑块背景色 */  
    min-height: 20px; /* 最小高度 */  
}  
  
QScrollBar::add-line:vertical {  
    border: none; /* 无边框 */  
    background: none; /* 无背景 */  
    height: 0px; /* 高度 */  
    subcontrol-position: bottom; /* 位置 */  
    subcontrol-origin: margin; /* 起始位置 */  
}  
  
QScrollBar::sub-line:vertical {  
    border: none; /* 无边框 */  
    background: none; /* 无背景 */  
    height: 0px; /* 高度 */  
    subcontrol-position: top; /* 位置 */  
    subcontrol-origin: margin; /* 起始位置 */  
}  
  
QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {  
    border: none; /* 无边框 */  
    background: none; /* 无背景 */  
    height: 0px; /* 高度 */  
}  
  
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {  
    background: none; /* 无背景 */  
}

</string>
                  </property>
                  <property name="editTriggers">
                   <set>QAbstractItemView::NoEditTriggers</set>
                  </property>
                  <property name="rowCount">
                   <number>10</number>
                  </property>
                  <attribute name="horizontalHeaderStretchLastSection">
                   <bool>true</bool>
                  </attribute>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <column>
                   <property name="text">
                    <string>教室</string>
                   </property>
                   <property name="textAlignment">
                    <set>AlignCenter</set>
                   </property>
                  </column>
                 </widget>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <widget class="QFrame" name="frame_text">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>500</width>
                 <height>600</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>500</width>
                 <height>600</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <spacer name="verticalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeType">
                     <enum>QSizePolicy::Fixed</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>10</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                     <spacer name="horizontalSpacer_2">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_2">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>150</width>
                        <height>50</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>微软雅黑</family>
                        <pointsize>20</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>问题反馈</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QTextEdit" name="textEdit"/>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <item>
                   <spacer name="verticalSpacer">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeType">
                     <enum>QSizePolicy::Fixed</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <item>
                     <spacer name="horizontalSpacer_10">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QPushButton" name="Btn_clean">
                      <property name="minimumSize">
                       <size>
                        <width>90</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>微软雅黑</family>
                        <pointsize>20</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>清空</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer_9">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QPushButton" name="Btn_send">
                      <property name="minimumSize">
                       <size>
                        <width>90</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>微软雅黑</family>
                        <pointsize>20</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>提交</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer_11">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <spacer name="verticalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeType">
                     <enum>QSizePolicy::Fixed</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_18">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_10">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../qrc.qrc"/>
 </resources>
 <connections/>
</ui>
